Library/Formula/mecab-ko.rb

```diff
@@ -2,14 +2,16 @@
 
 class MecabKo < Formula
   homepage "https://bitbucket.org/eunjeon/mecab-ko"
-  url "https://bitbucket.org/eunjeon/mecab-ko/downloads/mecab-0.996-ko-0.9.1.tar.gz"
-  version "0.996-ko-0.9.1"
-  sha1 "68dad4486d7b74fa5fe7aef0b440e00f17a9e59c"
+  url "https://bitbucket.org/eunjeon/mecab-ko/downloads/mecab-0.996-ko-0.9.2.tar.gz"
+  version "0.996-ko-0.9.2"
+  sha1 "4947620429dea49a69be84efc17736c4875c7180"
 
   # https://bitbucket.org/eunjeon/mecab-ko/pull-request/1/mecab-ko-ipadic-ipadic/diff
-  # Below comments out the dicdir path which produces runtime error
-  # and mecab-ko requires mecab-ko-dic rather than mecab-ipadic, unlike
-  # the original mecab.
+  # Upstream decided not to comment out the dicdir path but replaced
+  # 'ipadic' with 'mecab-ko-dic' instead in mecabrc file. 
+  # Though the change resolves the error of mecab-ko-dic path by source
+  # installation, it still doesn't fit with Homebrew as it's expected
+  # installed under /usr/local/ with mecab-ko-dic regardless of version.
```

Comment:
We need to try and work something out so we don't carry a patch. If upstream will never accept this patch as-is can we use `inreplace` instead?